test/series.test.js

```diff
@@ -20,7 +20,10 @@ import {
 
 describe('series:pageViews', () => {
   it('pageViews:bare', () => {
-    assert.equal(pageViews({ weight: 1 }), 1);
+    assert.equal(pageViews({ weight: 1, events: [] }), 1);
+    assert.equal(pageViews({ weight: 1, events: [{ checkpoint: 'prerender' }] }), 0);
```

Comment:
@kptdobe isn't this the test case you wanted?

Reply:
No, it is the other way around where there is a `prerender`. You do not change the data thus I assume you do not have a test with a `prerender` that would not be counted